src/core/numbers.cc

```diff
@@ -694,21 +694,21 @@ CL_DEFUN Number_sp contagen_div(Number_sp na, Number_sp nb) {
   case_Bignum_v_Complex : {
       Complex_sp cb = gc::As<Complex_sp>(nb);
       return complex_divide(clasp_to_double(na), 0.0,
-                            clasp_to_double(cb->real()), clasp_to_double(cb->imaginary()));
+                            clasp_to_double(gc::As<Number_sp>(cb->real())), clasp_to_double(gc::As<Number_sp>(cb->imaginary())));
```

Comment:
cb->real() and cb->imaginary() return Real_sp types.  These inherit from Number_sp - so don't use gc::As<Number_sp>(cb->real()).   

clasp_to_double(cb->real()) should be fine.
If you want to be explicit use clasp_to_double(gc::As_unsafe<Number_sp>(cb->real()))

Reply:
clasp_to_double(cb->real()) is not fine, the compiler is telling that type is ambigious and it doesn't know whether to call clasp_to_double for number or for T, so I will gc::As_unsafe<Number_sp> instead